testapi.pm

```diff
@@ -726,6 +726,14 @@ sub x11_start_program {
     return $distri->x11_start_program($program, $timeout, $options);
 }
 
+sub _handle_script_run_ret {
+    my ($ret, $cmd, %args) = @_;
```

Comment:
can we get even more info from $ret value ? like what was error code at least or something like this